hnn_core/network_builder.py

```diff
@@ -222,6 +222,14 @@ def _long_name(short_name):
     return short_name
 
 
+def _short_name(short_name):
```

Comment:
In line with @cjayb comments on `param` file checking, I'm starting to wonder if these sorts of conversions/processing should be moved into a member function of `Network`. Something like `validate_params()` that gets run at instantiation.

Reply:
I think this short name vs long name should go. We need a PR for this. See #148. I would stick to the short name everywhere because that's what the params file use.

Reply:
+1 on separate PR(s) on cleaning up params! #148 is a start.